packages/react/src/components/Slider/Slider.js

```diff
@@ -367,7 +367,7 @@ export default class Slider extends PureComponent {
     // If shift was held, account for the stepMultiplier
     if (evt.shiftKey) {
       const stepMultiplier =
```

Comment:
![image](https://user-images.githubusercontent.com/15822070/127675499-39466387-be67-4dd3-82f4-ead8abb4f4ec.png)

Looks like CircleCI is getting choked up around here.